actor/v2action/service_access.go

```diff
@@ -16,11 +16,17 @@ func (actor Actor) EnableServiceForAllOrgs(serviceName string) (Warnings, error)
 	}
 
 	for _, plan := range servicePlans {
-		warnings, err := actor.CloudControllerClient.UpdateServicePlan(plan.GUID, true)
+		warnings, err := actor.removeOrgLevelServicePlanVisibilities(plan.GUID)
```

Comment:
From what we can see, the legacy command did not perform this removal when making a plan public. What was the impetus for adding this step?

Reply:
Hi Tom – in the process of doing this work, we noticed that the old servicePlanVisibility objects weren't being swept away when a whole plan was made public. It seemed to us that these org-specific visibilities should be cleaned up in this case.

If this is incorrect behaviour, let us know and we're happy to remove it.

Reply:
I believe the legacy command did do this step. 
https://github.com/cloudfoundry/cli/blob/b6a3695a5db06d6628428067e0a78e63041ea1ca/cf/actors/services_plans.go#L182

So I think this change seems correct and inline with the pre-refactored code.

Reply:
@tjvman can you just confirm this resolves your concern when you get a chance?